infrastructure/docker/cover-service-jobs/etc/supervisor.d/search_queue.ini

```diff
@@ -0,0 +1,11 @@
+[program:search_queue]
```

Comment:
Do you have a supervisor job for ApiSearchQueue https://github.com/danskernesdigitalebibliotek/ddb-cover-service-importers/blob/develop/src/Service/VendorService/TheMovieDatabase/Queue/ApiSearchProcessor.php#L97

or should this be a part of importer Pod?

Reply:
Yes... the importers image is the base image for jobs.

Reply:
No as it has not been decided which and when imports should run... so it's done by hand until then.